plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -535,10 +555,38 @@ private ConnectorTableMetadata getTableMetadata(ConnectorSession session, Schema
             properties.put(PARTITIONING_PROPERTY, toPartitionFields(icebergTable.spec()));
         }
 
+        getParquetBloomFilterProperties(session, table, properties);
```

Comment:
Can we get this info from icebergTable similar to `getFileFormat(icebergTable)` ?